plugins/src/main/java/arm/org/tron/plugins/DbArchive.java

```diff
@@ -0,0 +1,46 @@
+package org.tron.plugins;
+
+import java.util.concurrent.Callable;
+import lombok.extern.slf4j.Slf4j;
+import org.tron.common.arch.Arch;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+
+/*
+  ARM platforms support RocksDB only; this command is not needed but retained for compatibility.
```

Comment:
--> "ARM platform only supports RocksDB, this command is not supported but retained for compatibility."